modules/datadict/php/datadict.class.inc

```diff
@@ -66,7 +66,7 @@ class Datadict extends \NDB_Menu_Filter
                               );
         $this->query        = " FROM parameter_type pt LEFT JOIN 
                                 parameter_type_override pto USING (Name)
-                 		WHERE pt.Queryable=1";
+                 		        WHERE pt.Queryable=1";
```

Comment:
lines not all indented equal.
Would also put the `LEFT JOIN` on the second line for clarity